utils/logger.ts

```diff
@@ -1,10 +1,21 @@
 import { logflarePinoVercel } from '@atdrago/pino-logflare';
 import pino from 'pino';
 
-const { send, stream } = logflarePinoVercel({
-  apiKey: process.env.NEXT_PUBLIC_LOGFLARE_API_KEY,
-  sourceToken: process.env.NEXT_PUBLIC_LOGFLARE_SOURCE_TOKEN,
-});
+const sendToLogflare =
+  !!process.env.NEXT_PUBLIC_LOGFLARE_API_KEY &&
+  !!process.env.NEXT_PUBLIC_LOGFLARE_SOURCE_TOKEN;
+
+const { send, stream } = sendToLogflare
+  ? logflarePinoVercel({
+      apiKey: process.env.NEXT_PUBLIC_LOGFLARE_API_KEY,
+      sourceToken: process.env.NEXT_PUBLIC_LOGFLARE_SOURCE_TOKEN,
+    })
+  : {
+      send: () => {
+        // noop
+      },
+      stream: process.stdout,
```

Comment:
I'm not sure this will work on the client, where `process.stdout` doesn't exist

Reply:
I think Pino only uses the `browser` section when running on the client.

Reply:
Ah okay, I forgot about that! So `process.stdout` on the client here would just be `undefined`, which isn't necessarily bad